src/examples/example-controls.scss

```diff
@@ -1,11 +1,19 @@
-@use '../style/functions';
+/**
+ * @prop --example-controls-max-columns-width: Defines maximum width of columns. Defaults to `6rem`.
+ * @prop --example-controls-column-layout: Defines how columns take up the available space. By default, it is set to be `auto-fill`, which fills the row with as many columns as it can. But it can also be set it to `auto-fit` stretches the columns to occupy all the available space.
+ */
 
-limel-flex-container {
-    margin-top: functions.pxToRem(20);
-    flex-wrap: wrap;
-}
+:host(limel-example-controls) {
+    display: grid;
+    grid-template-columns: repeat(
+        var(--example-controls-column-layout, auto-fill),
+        minmax(var(--example-controls-max-columns-width, 6rem), 1fr)
+    );
+    gap: 0 1rem;
+    align-items: center;
```

Comment:
> ![image](https://user-images.githubusercontent.com/35954987/210252595-cbf9e2e9-319b-4419-a0ba-e245a662c91b.png)
> 
> I don't know why, but the online preview version of the build is not showing the new component correctly ☝️ this is not how this is supposed to look like. I also had trouble to see changes on my local dev. I think you have to checkout the branch locally, remove the `dist` folder, and then `npm ci` and `npm start` to see the changes.

"The online preview version"? 🤔 You mean the docs for the PR? Those are identical to what will be published for the release, so if the component doesn't look right there, then it won't in the release either… 🙁

However, I think the problem is that you are targeting the `host` here, but the component doesn't use shadow-DOM. I don't know why it doesn't use shadow-DOM to begin with. You can try enabling that, or target `limel-example-controls` rather than `:host(limel-example-controls)`. (I haven't tried it myself yet, but I think that ought to work.)

Reply:
>  the problem is that you are targeting the host here,

🙈 yes, in a last fixup, I removed `shadow: true,`. When I was struggling to understand why it doesn't work and clean-reinstalled everything it seemed to me that it doesn't have any effect. So that's why. 

Now I added it back.